### PR TITLE
[DOCS] Removes redundant LDAP realm settings

### PR DESCRIPTION
**Pull request by lcawl**
This PR removes the redundant information about LDAP realm settings from
https://www.elastic.co/guide/en/elastic-stack-overview/master/ldap-realm.html
after integrating the missing information in
https://www.elastic.co/guide/en/elasticsearch/reference/master/security-settings.html#ref-ldap-settings

### CODE REVIEW

File: x-pack/docs/en/settings/security-settings.asciidoc [hunk 1]
```diff
@@ -218,44 +234,47 @@ The scope of the user search. Valid values are `sub_tree`, `one_level` or
...
-Specifies the filter used to search the directory in attempt to match
+Specifies the filter used to search the directory in attempts to match
 an entry with the username provided by the user. Defaults to `(uid={0})`.
 `{0}` is substituted with the username provided when searching.
 
 `user_search.attribute`::
-This setting is deprecated; use `user_search.filter` instead.
-The attribute to match with the username presented to. Defaults to `uid`.
+deprecated[5.6] Use `user_search.filter` instead.
+The attribute to match with the username presented to.
```
Comment (jaymode) [inline x-pack/docs/en/settings/security-settings.asciidoc]: good catch. Maybe `The attribute to match with the username sent with the request` or `username provided for authentication`

File: x-pack/docs/en/settings/security-settings.asciidoc [hunk 2]
```diff
@@ -265,30 +284,32 @@ Specifies whether the group search should be `sub_tree`, `one_level` or
 `base` specifies that the `base_dn` is a group object, and that it is the
 only group considered. Defaults to  `sub_tree`.
 
-`group_search.filter`::
+`group_search.filter`:: Specifies a filter to use to look up a group.
```
Comment (jaymode) [inline x-pack/docs/en/settings/security-settings.asciidoc]: should this be on a new line?

File: x-pack/docs/en/settings/security-settings.asciidoc [hunk 3]
```diff
@@ -265,30 +284,32 @@ Specifies whether the group search should be `sub_tree`, `one_level` or
...
-`group_search.filter`::
+`group_search.filter`:: Specifies a filter to use to look up a group. 
 When not set, the realm searches for `group`, `groupOfNames`, `groupOfUniqueNames`,
 or `posixGroup` with the attributes `member`, `memberOf`, or `memberUid`.  Any
 instance of `{0}` in the filter is replaced by the user attribute defined in
 `group_search.user_attribute`.
 
 `group_search.user_attribute`::
-Specifies the user attribute that will be fetched and provided as a parameter to
+Specifies the user attribute is fetched and provided as a parameter to
```
Comment (jaymode) [inline x-pack/docs/en/settings/security-settings.asciidoc]: maybe `that is`?

File: x-pack/docs/en/settings/security-settings.asciidoc [hunk 4]
```diff
@@ -369,27 +394,28 @@ See <<ssl-tls-settings,`xpack.ssl.verification_mode`>> for an explanation of
...
+communicating with the LDAP server. 
 Supported cipher suites can be found in Oracle's http://docs.oracle.com/javase/8/docs/technotes/guides/security/SunProviders.html[
 Java Cryptography Architecture documentation]. Defaults to the value of
 `xpack.ssl.cipher_suites`.
 
 `cache.ttl`::
-Specifies the time-to-live for cached user entries (a user and its credentials
-are cached for this period of time). Use the standard Elasticsearch
-{ref}/common-options.html#time-units[time units]). Defaults to  `20m`.
+Specifies the time-to-live for cached user entries. A user and its credentials
```
Comment (jaymode) [inline x-pack/docs/en/settings/security-settings.asciidoc]: Maybe we should say `a hash of its credentials` to more accurately represent what is being cached

**Review by jaymode — APPROVED** [review]
I left a couple of suggestions, other than those LGTM

**Comment by lcawl** [comment]
Thanks, @jaymode I've implemented those changes.

**Review by jaymode — APPROVED** [review]

**Pull request merged**